src/app/(content-pages)/transform-property/TransformPropertyPage.tsx

```diff
@@ -28,59 +51,303 @@ export default function TransformPropertyPage() {
           />
         </div>
 
-        <h2 className="heading-2xl font-bold mt-8">Types of Interventions</h2>
+        <h2 className="heading-2xl font-bold mt-8">Basic Interventions</h2>
         <p className="body-md">
           The original research that Clean & Green Philly is based on promotes
-          “cleaning and greening”. This includes removing trash, grading the
+          “cleaning and greening.” This includes removing trash, grading the
           land, planting trees, installing low fences, and maintaining the
           property. These are low-cost interventions and have been proven to be
           highly effective; even something as simple as cleaning up trash can
-          have a big impact. That said, there are many ways to transform a
-          vacant property. You could, for example:
-        </p>
-        <ul className="list-disc pl-6 my-4">
-          <li>Partner with a local restaurant to offer outdoor eating</li>
-          <li>Install bike parking</li>
-          <li>
-            Work with the Philadelphia Water Department to install green
-            stormwater infrastructure
-          </li>
-          <li>
-            Plant a pollinator garden to attract birds, butterflies, and bees
-          </li>
-          <li>
-            Connect with a non-profit organization to build affordable housing
-          </li>
-        </ul>
-        <p className="body-md">
-          These are just some of the many possibilities. Remember to be both
-          creative and practical: there is no one-size-fits-all solution, and
-          Philadelphia will need a variety of uses in its vacant properties,
-          including green space, housing, parks, commercial space, and more.
+          have a big impact.
         </p>
+        <div className="grid grid-cols-1 lg:grid-cols-3 md:grid-cols-1 gap-6 py-6">
+          <ContentCard
+            hasLink={true}
+            image={
+              <Image
+                src={cleanup}
+                alt={""}
+                placeholder="blur"
+                className="rounded-t-md w-full h-44 object-cover object-center"
+              />
+            }
+            title="Clean Garbage & Debris"
+            body="Organizing a community clean-up or hiring a waste removal service to clean a lot, enhances its appearance, lifts mental health, and eliminates potential hazards."
+            label="Cost:"
+            description="Low"
+            labelUpkeep="Upkeep:"
+            upkeepLevel="None"
+            links={[
+              {
+                url: "https://nkcdc.org/community/cleaning-greening/community-cleanup-resources/",
+                text: "Resources on NKCDC",
+              },
+            ]}
```

Comment:
instead of hardcoding `upkeepLabel` and `upkeepLevel`, would it possible to match the approach you make with the `links` array, and have a param of something like:
```
<ContentCard
...
details={[
  {
    label: "Cost"
    data: "Low"
  },{
    label: "Upkeep"
    data: "None"
  }
]}